documentation/src/main/asciidoc/topics/con_expiration_reaper.adoc

```diff
@@ -0,0 +1,24 @@
+[id='expiration_reaper-{context}']
+= Expiration Reaper
+{brandname} uses a reaper thread that runs periodically to detect and remove
+expired entries. The expiration reaper ensures that expired entries that are no
+longer accessed are removed.
+
+The {brandname} `ExpirationManager` interface handles the expiration reaper and
+exposes the `processExpiration()` method.
+
+In some cases, you can disable the expiration reaper and manually expire
+entries by calling `processExpiration()`; for instance, if you are using local
+cache mode with a custom application where a maintenance thread runs
+periodically.
+
+[IMPORTANT]
+====
+If you use clustered cache modes or cache stores, you should never disable the
```

Comment:
We may want to mention that currently you cannot disable the expiration reaper when a store is in use.

Reply:
We may want to revisit this requirement, I can't quite remember why we did this. It was over almost 5 years ago :)